### PR TITLE
Initial generation of nav json

### PR DESCRIPTION
**Pull request by drewpotter**
Here is the nav json as requested.

### CODE REVIEW

File: app/models/event.rb [hunk 1]
```diff
@@ -22,6 +22,14 @@ def self.filters_to_json
     ].to_json
   end
 
+  def self.years_to_json
+    events = Event.all.order(start_date: :asc)
```
Comment (Levia) [inline app/models/event.rb]: If you are ordering in the two lines below, you don't need to sort here as well.

File: app/models/event.rb [hunk 2]
```diff
@@ -22,6 +22,14 @@ def self.filters_to_json
...
+    start_year = events.pluck(:start_date).compact.sort.first.year
+    end_year = events.pluck(:start_date).compact.sort.last.year
```
Comment (Levia) [inline app/models/event.rb]: I think it would make more sense to get the `end_date` here instead of `start_date`

File: app/models/event.rb (hunk 1 above)
Comment (drewpotter) [reply app/models/event.rb]: Thanks @Levia I will fix this!

File: app/models/event.rb (hunk 2 above)
Comment (drewpotter) [reply app/models/event.rb]: Ok thanks @Levia I will make that change!

**Pull request merged**